src/riak_cs_config.erl

```diff
@@ -402,6 +403,15 @@ read_before_last_manifest_write() ->
 region() ->
     get_env(riak_cs, region, ?DEFAULT_REGION).
 
+%% @doc This switch changes mapreduce input from listkeys to
+%% fold_objects after riak_kv's mapreduce input optimization
+%% merged and enabled. With fold_objects it will have 1/2 IO
+%% when scanning the whole bucket.
+-spec use_2i_for_storage_calc() -> boolean().
+use_2i_for_storage_calc() ->
+    riak_cs_list_objects_utils:fold_objects_for_list_keys()
```

Comment:
The deprecation plan is to mention in 2.0 release notes and remove the code later, like late 2.0.x or 2.1. That's why I still hold this config.